docs/dictionary/command/constant.lcdoc

```diff
@@ -14,18 +14,18 @@ OS: mac, windows, linux, ios, android
 Platforms: desktop, server, mobile
 
 Example:
-constant defaultName="Jones"
+constant kDefaultName="Jones"
 
 Example:
-constant entryA="EF9993333WX786",entryB="GJ773281YX342"
+constant kEntryA="EF9993333WX786",kEntryB="GJ773281YX342"
 
 Parameters:
-constantsList (enum):
-The constantsList consists of one or more name=value pairs, separated by
-commas: 
+constantsList:
+The constantsList consists of one or more <name>=<value> pairs, 
```

Comment:
As they aren't strictly parameters or links, the angle brackets won't work. You could do it with `&lt;` and `&gt;` though. I think we quite often use angle brackets for placeholders in the docs, so I think the change does make sense.